DiscountDiscover/Views/DealCell.m

```diff
@@ -9,8 +9,20 @@
 #import "DealCell.h"
 #import "UIImageView+AFNetworking.h"
 
+@interface DealCell ()
+
+@property (nonatomic) BOOL cardStyled;
+
+@end
+
 @implementation DealCell
 
+- (void)awakeFromNib {
+    [super awakeFromNib];
+    
+    self.cardStyled = NO;
```

Comment:
I think you can call `[self cardStyling]` here directly and avoid the boolean flag

Reply:
I tried that, but the shadows were off, and I believe it's because the cells are a different size before the content is added due to auto layout (?). But if there's still a better way to do it, let me know!

Reply:
That's fine then, no worries